modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -45,13 +41,18 @@ To create an API asset:
 . In Exchange, select *Publish new asset*.
 . Enter the asset portal name.
 . Select the asset type from the drop-down list:
-+
 
 * REST API - RAML: Provide a RAML API specification file. RAML specifications must be a RAML file (.raml).
 * REST API - OAS: Provide an OAS API specification file. OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
 * SOAP API - WSDL: Provide a WSDL API specification file. SOAP specifications file can be either a WSDL (.wsdl) or XML (.xml) file.
 * AsyncAPI: Provide an AsyncAPI specification file in a YAML (.yaml) file.
-* API Spec Fragment - RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml).
+* API Spec Fragment:
++
+RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml). 
++
+JSON: Provide an API Fragment JSON specification (.json) file. 
++
+OpenAPI Specification (OAS): Provide an API Fragment OpenAPI specification file either JSON or YAML.
```

Comment:
```suggestion
* API Spec Fragment - RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml). 
* API Spec Fragment - JSON: Provide an API Fragment JSON specification (.json) file. 
* API Spec Fragment - OpenAPI Specification (OAS): Provide an API Fragment OpenAPI specification file either JSON or YAML.
```

Reply:
To maintain consistency with the text above we could use the same structure.

Reply:
agreed